yarn.lock

```diff
@@ -3110,7 +3110,7 @@ file-entry-cache@^2.0.0:
     flat-cache "^1.2.1"
     object-assign "^4.0.1"
 
-file-loader@^1.1.10:
```

Comment:
If the yarn.lock is updated, probably we should update `package.json`  in order to have them synced.

Reply:
Jamie tracked this back to a commit by Rupert, I think this commit will bring yarn.lock back to where it should be.